app/views/layouts/application.html.haml

```diff
@@ -26,27 +26,21 @@
 
     %title= html_title
 
-    = javascript_pack_tag 'common', crossorigin: 'anonymous'
+    = stylesheet_pack_tag 'flavours/vanilla/common', media: 'all', crossorigin: 'anonymous' # upstream uses `common` but that's implicitly defined
+    = stylesheet_pack_tag current_theme, media: 'all', crossorigin: 'anonymous'
 
     -# Needed for the wicg-inert polyfill. It needs to be on it's own <style> tag, with this `id`
-    = stylesheet_pack_tag 'core/inert', media: 'all', id: 'inert-style'
+    = stylesheet_pack_tag 'flavours/vanilla/inert', media: 'all', id: 'inert-style'
```

Comment:
Is loading vanilla inert instead of the one of the current flavour really desired here?
This also applies to some other views.
For example my fork compiles without the vanilla flavour, so this causes webpack errors.
Not something I can't deal with downstream, but all flavours include inert already.

Reply:
hm, I did that because those CSS packs are extremely unlikely to change, and there are places (mailer, error) where we don't necessarily have a current flavour set.

I hadn't thought about forks that would not have the default flavor available, I'll give it some thought.

Reply:
Thank you
I know it's somewhat of a special use-case/edge-case.
Yeah, I ran into the issue with mailer while testing this.
I opted for `Setting.default_settings['flavour]` instead of `current_flavour` for now.

Reply:
I believe the last commit I pushed should address this.